components/org.apache.stratos.manager/src/main/database/Stratos-dbScript

```diff
@@ -0,0 +1,91 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
```

Comment:
It would be better to rename this file to something similar stratos.sql. Please refer existing database files.
Thanks